### PR TITLE
bpo-33859: Fix spelling mistakes in docs

### PR DESCRIPTION
**Pull request by tirkarthi**
I have found some typos in docs folder using aspell. I have fixed them. The changes are as below : 

* Doc/library/codecs.rst - cypher - cipher (Reverted)
* Doc/library/email.rst - Protcol - Protocol
* Doc/library/importlib.rst - abstact - abstract
* Doc/library/xmlrpc.client.rst - unmarsalling - unmarshalling
* Doc/license.rst - aheared to - adhered to (Reverted)
* Doc/using/cmdline.rst - descibed - described
* Doc/whatsnew/3.3.rst - accumlated - accumulated
* Doc/whatsnew/3.6.rst - Lollilop - Lollipop
* Doc/whatsnew/3.7.rst - direcory - directory

Find typos : 

`find . -iname '*rst' | xargs -I{} sh -c "aspell --master=en_US --extra-dicts=en_GB --ignore 3 list < {}" | sort | uniq > typos.txt`

Ignore case and manual search

`tr '[:upper:]' '[:lower:]' < typos.txt | sort | uniq | less`

This requires manually looking at output of less and then making changes.

https://bugs.python.org/issue33859

### CODE REVIEW

**Comment by the-knights-who-say-ni** [comment]
Hello, and thanks for your contribution!

I'm a bot set up to make sure that the project can legally accept your contribution by verifying you have signed the [PSF contributor agreement](https://www.python.org/psf/contrib/contrib-form/) (CLA).

Unfortunately our records indicate you have not signed the CLA. For legal reasons we need you to sign this before we can look at your contribution. Please follow [the steps outlined in the CPython devguide](https://devguide.python.org/pullrequest/#licensing) to rectify this issue.


When your account is ready, please add a comment in this pull request
and a Python core developer will remove the `CLA not signed` label
to make the bot check again.

Thanks again for your contribution, we look forward to reviewing it!

**Comment by tirkarthi** [comment]
I have signed the CLA with bugs.python.org 

username: xtreak

**Comment by tirkarthi** [comment]
Please add "skip news" label since they are typo fixes and I think it's not necessary. Kindly correct me if I am wrong since I am a first time contributor.

Thanks

File: Doc/library/codecs.rst
Comment (willingc) [inline Doc/library/codecs.rst]: Let's leave this unchanged since `cypher` is a valid word.

**Review by willingc — COMMENTED** [review]
Hi @tirkarthi, Thanks for your first contribution. I recommend leaving the cypher as is in codecs.rst.

File: Doc/library/codecs.rst
Comment (tirkarthi) [reply Doc/library/codecs.rst]: Thanks, I took reference from https://en.m.wikipedia.org/wiki/Caesar_cipher and I think I couldn't find Cypher across the docs making the change. I will revert it.

Comment (tirkarthi) [reply Doc/library/codecs.rst]: @willingc kindly confirm if the change  needs to be reverted given the above link.

Thanks

Comment (willingc) [reply Doc/library/codecs.rst]: @tirkarthi Both are used interchangeably in the literature. I would leave the original author's spelling. Thanks.

Comment (tirkarthi) [reply Doc/library/codecs.rst]: @willingc Sure. Reverted to cypher. Thanks.

File: Doc/license.rst
Comment (serhiy-storchaka) [inline Doc/license.rst]: You need a special authorization from PSF lawyers for changing the text of the license.

I suggest to leave this text as is.

Comment (tirkarthi) [reply Doc/license.rst]: Reverted it back. Thanks.

**Review by serhiy-storchaka — APPROVED** [review]

File: Doc/license.rst
Comment (willingc) [reply Doc/license.rst]: Great catch @serhiy-storchaka. I missed the license as we typically have in all caps on Jupyter stuff.

**Comment by terryjreedy** [comment]
There is an ???@python.org email for legal issues.  It should be in the license or somewhere on python.org.  Sending an email about the license type might get it fixed.

What's New in 3.7 does not exist in 3.6 and should block automerging to 3.6.  I personally would put it in a separate PR for this issue only backported to 3.7.  Or you can manually do the 3.6 backport.

**Comment by terryjreedy** [comment]
For this issue, it may be that some of the misspellings were introduced in 3.7 or 3.8, so that backports will fail anyway.  (I am speculating this because there was a general fix-typos issue just a few months ago.)  An alternative to manual backports would be to runs the checks on 3.6 and 3.7 as well as 3.8.

**Comment by tirkarthi** [comment]
Thanks for the feedback. Sorry, I am a little confused since I am a beginner. I suppose there is a bot named @miss-islington that would try them and is there a way to manually trigger it to see if the backports fail? With respect to backporting can you please point me to docs so that I can learn more about this. It will be helpful if you can give me source branches for these so that I can try making separate PRs by running checks on them to see if the spellings are fixed and keep this one to master as such. 

It will be helpful to know merge timelines for this PR and others like 3.7 since 3.8 alpha is the latest one with 3.7 in RC phase. I will try these over the weekend.

Thanks.

**Comment by tirkarthi** [comment]
I think cherry-picker.py will handle the branches but needs this PR to be merged so that I can use the squashed commit. I think I can wait for this to be merged. I am open to inputs on this.

Thanks

**Comment by serhiy-storchaka** [comment]
In any case we should merge a PR into the master branch first than create backporting PRs.

**Comment by serhiy-storchaka** [comment]
As you can see, the @miss-islington bot successfully created a backport for 3.7, but failed to create a backport to 3.6 because of changes to `Doc/whatsnew/3.7.rst`. She/it left a suggestion for manual backporting.

**Comment by tirkarthi** [comment]
@serhiy-storchaka Thanks, I will raise a separate PR as a manual backport.

**Pull request merged**
